cluster/src/main/java/com/hartwig/pipeline/reruns/NoopPersistedDataset.java

```diff
@@ -0,0 +1,18 @@
+package com.hartwig.pipeline.reruns;
+
+import java.util.Optional;
+
+import com.hartwig.pipeline.datatypes.DataType;
+import com.hartwig.pipeline.metadata.RunMetadata;
+
+public class NoopPersistedDataset implements PersistedDataset {
```

Comment:
Maybe it's just a nit but I do think this would be easier to read as `NoOpPersistedDataset`.